.github/workflows/scan.yml

```diff
@@ -68,8 +71,8 @@ jobs:
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          # TODO: see https://docs.sonarsource.com/sonarcloud/advanced-setup/analysis-parameters/
-          # for parameters to configure
           args: >
             -Dsonar.organization=${{ github.repository_owner }}
             -Dsonar.projectKey=${{ github.repository_owner }}_${{ github.event.repository.name }}
+            -Dsonar.sources=app/src/main,app/src/debug
+            -Dsonar.tests=app/src/test,app/src/androidTest
```

Comment:
🎨 Like in other repos we should perform all the configuration in one place, either here or in the Gradle config; seems like the latter is what's needed.

Reply:
Done and noted

Reply:
I moved the source and test paths over to the gradle build script, but I didn't see this in the sonar documentation, so hopefully they belong there. I guess it's the same as including them as arguments.